proto/vtctldata.proto

```diff
@@ -988,6 +1034,13 @@ message UpdateThrottlerConfigRequest {
   bool check_as_check_shard = 8;
   // ThrottledApp indicates a single throttled app rule (ignored if name is empty)
   topodata.ThrottledAppRule throttled_app = 9;
+  // MetricName is the name of the metric for which we apply a new threshold
```

Comment:
SImilar question as above: should we be able to specify a list of metrics here if multiple metrics are used or is each metric updated separately?

Reply:
You run one `vtctldclient UpdateThrottlerConfig` command, or send one `UpdateThrottlerConfig` for any metric you wish to apply a threshold to, like so:
```sh
$ vtctldclient UpdateThrottlerConfig --metric-name "loadavg" --threshold "2.5" commerce
```